specs/altair/p2p-interface.md

```diff
@@ -74,6 +74,15 @@ New topics are added in Altair to support the sync committees and the beacon blo
 
 The specification around the creation, validation, and dissemination of messages has not changed from the Phase 0 document.
 
+The derivation of the `message-id` has changed starting with Altair to incorporate the message topic along with the message data.
+The `message-id` MUST be the following 20 byte value computed from the message:
+* If `message.data` has a valid snappy decompression, set `message-id` to the first 20 bytes of the `SHA256` hash of
+  the concatenation of `MESSAGE_DOMAIN_VALID_SNAPPY` with the snappy decompressed message data and the topic name,
+  i.e. `SHA256(MESSAGE_DOMAIN_VALID_SNAPPY + snappy_decompress(message.data) + message.topic)[:20]`.
```

Comment:
It would be better to separate the message topic from the data bytes, or otherwise we could get weird collisions (start of topic name being embedded in the message data as suffix instead of the topic ID as prefix).
I suggest we order it like:
1. snappy domain
2. length of topic byte string
3. topic byte string
4. message data (decompressed or not, depending on snappy domain)

Reply:
yep, good idea! let me update